base_report_to_printer_label/models/res_users.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    default_label_printer_id = fields.Many2one(
+        comodel_name="printing.printer", string="Default Label Printer"
+    )
+
+    @api.model
+    def _register_hook(self):
+        super()._register_hook()
+        self.SELF_WRITEABLE_FIELDS.extend(["default_label_printer_id"])
+        self.SELF_READABLE_FIELDS.extend(["default_label_printer_id"])
+
+    def get_delivery_label_printer(self):
```

Comment:
The previous implementation was
```suggestion
    def get_delivery_label_printer(self, delivery_type):
```
but this parameter was not used at all, better to remove it.